packages/@atjson/offset-annotations/src/annotations/cneaudio-embed.ts

```diff
@@ -0,0 +1,9 @@
+import { BlockAnnotation } from "@atjson/document";
+
+export class CneAudioEmbed extends BlockAnnotation<{
+  url: string;
+  targetId: string;
```

Comment:
We'll want `anchorName` here for consistency. I think we should consider using that attribute rather than `targetId` for consistency. (`anchorName` allows editors to jump to link to a specific part of the page using hashtags in URLs)

Reply:
Thanks Tim, we are using now `anchorName` insted of `targetId`

Reply:
I want to confirm though that the anchor name should not be used for the ID attribute on the div. The client should generate unique IDs at render time rather than have it stored as part of the data.

Reply:
hey @bachbui. Unless I am wrong, it is what I am doing here:
https://github.com/CondeNast/atjson/pull/1681/files#diff-a014cc398db9c717481da7d06790fb6936c7d3bca83f9404449fd993f6c2b409R152

Do you meant just remove it there and generate a div id directly in the render method?
https://github.com/CondeNast/atjson/pull/1681/files#diff-6e2df2e9de1bed7f97ce3e0e3871f7c1c140be23856875897b27baedd466db16R264

Thanks!